CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The changes are relative to the previous release, unless the baseline is specifi
 * Renamed AVIF_ENABLE_EXPERIMENTAL_METAV1 to AVIF_ENABLE_EXPERIMENTAL_MINI and
   updated the experimental reduced header feature to the latest specification
   draft.
+* Ignore gain maps with unsupported metadata.
```

Comment:
We should also mention we handle writer_version > 0 correctly. (The previous remaining data check would fail if new fields were added in writer_version > 0.) This can be done in a follow-up PR.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2382.